mlflow/utils/cli_args.py

```diff
@@ -156,7 +156,11 @@ def _resolve_env_manager(ctx, _, env_manager):
 )
 
 TIMEOUT = click.option(
-    "--timeout", "-t", help="Timeout in seconds to serve a request (default: 60)."
+    "--timeout",
+    "-t",
+    envvar="MLFLOW_SCORING_SERVER_REQUEST_TIMEOUT",
```

Comment:
This timeout is used in other CLIs such as `mlflow server`, which are not related to Model serving. Can we instead make the change in https://github.com/mlflow/mlflow/blob/96f861b9f32a8e6ad2473db3b3272a50f5f8e71a/mlflow/models/cli.py#L31?

Reply:
It's the same one I believe :) . `cli_args.TIMEOUT` references the other one. and semantically it's all the same "timeout for the scoring server"

But note, since this isn't passed into the spark_udf implementation, I've added a direct reference to the env var there also

Reply:
My mistake - this is, indeed the same arg. Thanks @alonisser !